puma/tests/test_histogram.py

```diff
@@ -390,14 +390,14 @@ def test_draw_vlines_histogram(self):
         )
         hist_plot.add(self.hist_1, reference=True)
         hist_plot.draw_vlines(
-            vlines_xvalues=[1, 2, 3],
+            xs=[1, 2, 3],
         )
         hist_plot.draw()
 
         plotname = "test_draw_vlines_histogram.png"
         hist_plot.savefig(f"{self.actual_plots_dir}/{plotname}")
         # Uncomment line below to update expected image
-        # hist_plot.savefig(f"{self.expected_plots_dir}/{plotname}")
+        hist_plot.savefig(f"{self.expected_plots_dir}/{plotname}")
```

Comment:
I think that this line and similar lines below should be commented

Reply:
Good catch, fixed